src/index.js

```diff
@@ -54,6 +54,11 @@ export default class RabbotClient {
     this.client = rabbot;
   }
 
+  // Function should recieve a queue message and return a gb-services style context.
+  setContextFunction(contextFunction) {
```

Comment:
Definitely. How does that work with hydration though? How can I pass the method which will live on gb-services to configured-rabbitmq-client at configuration time?